packages/core/src/external/quest/file/file-names.ts

```diff
@@ -1,22 +1,76 @@
+import { MetriportError } from "@metriport/shared";
 import { buildDayjs } from "@metriport/shared/common/date";
 
+const RESPONSE_FILE_PREFIX = "Metriport_";
+const RESPONSE_FILE_EXTENSION = ".txt";
+const DATE_ID_REGEX = /^(\d{8}|\d{12})$/;
+
 export function buildRosterFileName() {
   const dateId = buildDayjs().format("YYYYMMDD");
-  return `METRIPORT_${dateId}.txt`;
+  return `Metriport_roster_${dateId}.txt`;
```

Comment:
Changed to avoid confusion with the response files, which have a similar naming pattern.